capa/main.py

```diff
@@ -189,57 +189,32 @@ def has_rule_with_namespace(rules, capabilities, rule_cat):
     return False
 
 
+def is_internal_rule(rule):
+    return rule.meta.get("namespace", "").startswith("internal/")
```

Comment:
Maybe we can use `capa` instead of `internal`? I'm just thinking that may be clearer when browsing the rules. I'm fine with `internal` as well though.

Reply:
regardless, we can move `lib` and `doc` then to this path

Reply:
opinions not facts...

> Maybe we can use capa instead of internal

the directory is already in the `capa-rules` repository, so "capa" seems to duplicate the project name? maybe "internal" is too implicit about which project this is? i dont really care either way.

> regardless, we can move lib ... to this path

i don't think that lib rules are internal rules - they are meant for other people to re-use, too. therefore, i would prefer that we keep `lib` separate.

> regardless, we can move ... doc then to this path

this is fine with me, though ill raise the minor issue of mixing rules and not-rules in the same directory. probably not important.

Reply:
let's keep it as is